src/libs/fileDownload/FileUtils.ts

```diff
@@ -330,6 +331,21 @@ const resizeImageIfNeeded = (file: FileObject) => {
     }
     return getImageDimensionsAfterResize(file).then(({width, height}) => getImageManipulator({fileUri: file.uri ?? '', width, height, fileName: file.name ?? '', type: file.type}));
 };
+
+const createFile = (file: File): FileObject => {
+    if (getPlatform() === CONST.PLATFORM.WEB) {
+        return new File([file], file.name, {
+            type: file.type,
+            lastModified: file.lastModified,
+        });
+    }
+    return {
+        uri: file.uri,
+        name: file.name,
+        type: file.type,
+    };
+};
```

Comment:
```suggestion
const createFile = (file: File): FileObject => {
    if (getPlatform() === CONST.PLATFORM.ANDROID || getPlatform() === CONST.PLATFORM.IOS) {
        return {
            uri: file.uri,
            name: file.name,
            type: file.type,
        };
    }
    return new File([file], file.name, {
        type: file.type,
        lastModified: file.lastModified,
    });
};
```

Shouldn't it be like this? Pretty sure it won't work on desktop. I'm also confused why we need this function. Is there the same issue in other places where we use `AvatarWithImagePicker`?

> It was causing issues on native app because in native the file object requires different properties.

I see in some places that we set the `uri` property (which is required) like this, perhaps we need to do something similar

https://github.com/Expensify/App/blob/fcd16e4f45f1fb119ced7786114a33363f757849/src/pages/iou/request/step/IOURequestStepScan/index.tsx#L796